Changelog.org

```diff
@@ -1,3 +1,8 @@
+* 0.15
```

Comment:
Sorry, I should have said it before: Can you update it to Unreleased instead ?

Reply:
Sure! Sorry, should have looked into the git log how you did that before.